packages/client/utils/makeCheckinGreeting.ts

```diff
@@ -175,7 +175,127 @@ const questions = [
+  'If you had a magic button on your desk to bring you whatever you wanted instantly, what would it summon?',
+  'What’s your guilty pleasure right now?',
+  'What was the last thing you did that gave you child-like joy?',
+  'What’s a book you would like to live in?',
+  'What’s the last thing you did for the first time?',
+  'What’s one activity that grants you pure escapism?',
+  'You’re on a desert island and can bring one luxury with you – what is it?',
+  'What’s one characteristic you admire in others and why?',
+  'What’s an organization or private company you admire and why?',
+  'What was the first concert or gig you ever attended?',
+  'What does friendship mean to you?',
+  'What’s one possession or tool that enhances your daily life?',
+  'What smell or image triggers nostalgia for you?',
+  'How do you like to sit in a chair?',
+  'What’s a cause or purpose you deeply believe in?',
+  'What’s a hobby or side project you’ve been meaning to take up?',
+  'What ice cream flavour sums up your personality today?',
+  'What’s one thing your parents or guardians always nagged you about?',
+  'What’s something you do to relieve anger or stress?',
+  'What’s a sauce, condiment or dressing you couldn’t live without?',
+  'What are you doing when you feel most alive?',
+  'Tell us about a stranger you still remember. What made them memorable?',
+  'What would you do if you came home and found a penguin in your freezer?',
+  'What’s one thing you wish you’d been taught in school?',
+  'You’ve been granted a one-way ticket to another country of your choice. Where are you going?',
+  'What does it feel like to be you right now?',
+  'How have you changed in the last year?',
+  'What’s a food that reminds you of childhood?',
+  'What’s something our team is good at?',
+  'What’s something you’ve drawn inspiration from recently?',
+  'What’s something you like to do whenever you travel to a new country?',
+  'Who on the team would you like to swap jobs with for a day?',
+  'It’s your first day on the job as an evil villain. What’s the first piece of mischief you commit?',
+  'Who’s an ex-colleague from this job or another that you still think about?',
+  'What’s an adult problem nobody prepared you for?',
+  'What do your family think you do for work?',
+  'What’s a book that affected you so deeply (emotionally or intellectually) that you still regularly think about it?',
+  'What’s something that doesn’t deserve the hate it gets?',
+  'What’s a simple life hack that everyone can benefit from?',
+  'What is a lesson you feel you learned too late in life?',
+  'What’s your spirit animal this week?',
+  'If buying groceries were a game, what would be one of the loading screen tips?',
+  'What’s one of your recent pet peeves?',
+  'How would your best friend describe you?’,
+  'Congratulations! You’ve been chosen to represent your country in a global competition. What sport or activity are you doing?'
```

Comment:
Almost there! It looks like Typecheck is complaining about line 298. I think the quote mark at the end of line 297 is causing the mischief